aidants_connect_web/templates/layouts/nav.html

```diff
@@ -9,26 +9,26 @@
       <ul class="nav__links">
       {% if request.user.is_authenticated %}
         <li class="nav__item">
-          <a class="button-outline secondary" href="{% url 'dashboard' %}">Espace Aidant</a>
+          <a {% if request.resolver_match.view_name in 'dashboard,usagers,usager_details,usagers_mandats_autorisations_cancel_confirm' %}class="font-weight-bold"{% endif %} href="{% url 'dashboard' %}">Mon espace Aidant</a>
```

Comment:
Ça marche comme ça, et le coup du `in` avec une chaîne est _rusé_ 😄 
Mais je me demande si on ne pourrait pas faire plus _DRY_ et un peu plus "propre" aussi.

Pour le _DRY_, je te proposerais bien d'utiliser [`with`](https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#with), par exemple en encapsulant l'essentiel du code du template dans un bloc du genre : 
```
{% with view_name=request.resolver_match.view_name %}
...
{% endwith %}
```

Reply:
j'ai fait la modif avec le with_name. Tu me diras si tu espérais d'avantage de changement ou si c'est ce que tu imaginais